docs/network protocol.md

```diff
@@ -191,6 +191,21 @@ Sent to clients after a client requested this message be sent to them, more info
 ### InvalidPacket
 Sent to clients if the server caught a problem with a packet. This only occurs for errors that are explicitly checked for.
 
+#### Arguments
+| Name | Type | Notes |
+| ---- | ---- | ----- |
+| type | str | The [PacketProblemType](#PacketProblemType) that was detected in the packet |
+| original_cmd | Optional[str] | the `cmd` argument of the faulty packet, will be `None` if the `cmd` failed to be parsed |
+| text | str | A descriptive message of the problem at hand |
```

Comment:
```suggestion
| text | str | A descriptive message of the problem at hand. |
```

Reply:
^ that's not a complete sentence (which is why i suggested removing the 3 words on the front)